linker/src/main/kotlin/com/openlattice/linking/controllers/RealtimeLinkingController.kt

```diff
@@ -77,4 +88,59 @@ class RealtimeLinkingController(
 
         return matchedEntityPairs
     }
+
+    @RequestMapping(
+            path = [RealtimeLinkingApi.BLOCKING],
+            method = [RequestMethod.POST],
+            produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    override fun block(
+            @RequestBody blockingRequest: BlockingRequest
+    ): Map<UUID, Map<UUID, List<BlockedEntity>>> {
+        ensureAdminAccess()
+        val propertyFqnCache = CacheBuilder
+                .newBuilder()
+                .maximumSize(8192)
+                .build(CacheLoader.from { propertyTypeId: UUID? -> edmService.getPropertyTypeFqn(propertyTypeId!!) })
+
+        return blockingRequest.entities.mapValues { (entitySetId, entityKeyIds) ->
+            entityKeyIds.associateWith { entityKeyId ->
+                blocker
+                        .block(EntityDataKey(entitySetId, entityKeyId),top = blockingRequest.blockSize)
+                        .entities
+                        .map { (edk, entity) ->
+                            BlockedEntity(
+                                    edk,
+                                    entity.mapKeys { propertyFqnCache[it.key]!! }
+                            )
+                        }
+            }
+        }
+    }
+
+    override fun createNewLinkedEntity(entityDataKeys: MutableSet<EntityDataKey>?): UUID {
+        TODO("Not yet implemented")
+    }
+
+    override fun getLinkedEntityKeyIds(linkingEntitySetId: UUID?): MutableMap<UUID, MutableSet<EntityDataKey>> {
+        TODO("Not yet implemented")
+    }
+
+    override fun setLinkedEntities(
+            linkingEntitySetId: UUID?, linkedEntityKeyId: UUID?, entityDataKeys: MutableSet<EntityDataKey>?
+    ): Int {
+        TODO("Not yet implemented")
+    }
+
+    override fun addLinkedEntities(
+            linkingEntitySetId: UUID?, linkedEntityKeyId: UUID?, entityDataKeys: MutableSet<EntityDataKey>?
+    ): MutableSet<EntityDataKey> {
+        TODO("Not yet implemented")
+    }
+
+    override fun removeLinkedEntities(
+            linkingEntitySetId: UUID?, linkedEntityKeyId: UUID?, entityDataKeys: MutableSet<EntityDataKey>?
+    ): MutableSet<EntityDataKey> {
+        TODO("Not yet implemented")
```

Comment:
We've recently been adopting a pattern of every TODO needs a ticket number with it. Up to @UnsungHero97 whether to continue that here.